src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.nuspec

```diff
@@ -52,6 +52,7 @@
     <!-- uap10.0 -->
     <file src="uap10.0.16299\build\uwp\MSTest.TestAdapter.props" target="build\uap10.0\MSTest.TestAdapter.props" />
     <file src="uap10.0.16299\build\uwp\MSTest.TestAdapter.targets" target="build\uap10.0\MSTest.TestAdapter.targets" />
+    <file src="uap10.0.16299\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\uap10.0\" />
```

Comment:
I'm curious why is this needed?

Reply:
Yep this should not be needed except if we have an issue similar to netfx with assembly resolver but it'd be surprising we don't have it for other net/netcore.

Reply:
I think this was a bad merge then. Looks like Sergio's version had it - [I see it back in November.](https://github.com/microsoft/testfx/blob/2e89f20f8690f37484d121dd452716e2dc975d19/src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.nuspec). Will remove it.